h/views/api/config.py

```diff
@@ -4,6 +4,12 @@
 from h.util import cors
 
 
+class APIConfigError(Exception):
```

Comment:
Tossed this custom Exception in here but have zero attachment to the methodology of raising in this module. Looking for opinions. /cc @seanh

Reply:
I guess this would only ever be raised if the code was wrong (an `@api_config` had an invalid version number) and it would be raised at application startup? So code that raises this would never get committed and deployed (since the app would be failing to even start up).

In that case I think you could probably just raise `ValueError` instead of a custom exception.

If you still want a custom exception (for example because it makes writing tests easier) I think it should go in `h/views/api/exceptions.py` just for the sake of following the pattern of putting exceptions into `exceptions.py`. It'd make sense as a `ValueError` subclass to rather than just an `Exception` subclass, but it doesn't matter much.